database/sql/V075__add_call_data_to_quotes_and_order_quotes.sql

```diff
@@ -0,0 +1,37 @@
+-- This migration script is reversible.
+
+-- Step 1: Add a new column to the quotes table
+ALTER TABLE quotes 
+    ADD COLUMN verified boolean;
+
+-- Step 2: Add two new columns to the order_quotes table
+ALTER TABLE order_quotes 
+    ADD COLUMN verified boolean;
+
+-- Step 3: Create table with quote interactions
+CREATE TABLE quote_interactions (
+    quote_id bigint NOT NULL,
+    index int NOT NULL,
+    target bytea NOT NULL,
+    value numeric(78,0) NOT NULL,
+    call_data bytea NOT NULL,
+
+    PRIMARY KEY (quote_id, index)
```

Comment:
How about using DB to clean up this table automatically since the `quotes` table gets cleaned up periodically?
```suggestion
    PRIMARY KEY (quote_id, index)
    FOREIGN KEY (quote_id) REFERENCES quotes(id) ON DELETE CASCADE
```

Reply:
Implemented that.